src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.js

```diff
@@ -30,11 +30,13 @@ type Props = {
     getMentionWithQuery?: (searchStr: string) => void,
     getUserProfileUrl?: GetProfileUrlCallback,
     isCurrentVersion: boolean,
+    isResolvingEnabled?: boolean,
```

Comment:
Can we control this server-side so that `can_resolve` is always false if the feature is disabled?

Reply:
It is meant to based on `hasReplies` prop from ActivitySidebar. For server-side decision handling, we would need to send this data anyway

Reply:
I don't know if that really answered the question. Replies and resolve aren't directly related, they just happen to be being built at the same time. Can we make the decision about showing these elements based solely on the `can_resolve` permission, instead?

Reply:
It would help simplify some logic where `isResolvingEnabled && canResolve` is being used. How is `isResolvingEnabled` determined when this component is called?

Reply:
That idea make sense - can_resolve permission will be handled server-side 👍